docs/source/routing/about-router.mdx

```diff
@@ -80,6 +80,14 @@ You can choose to manage the runtime infrastructure for your routers by yourself
   class="hidden dark:block"
 />
 
+<Caution>
+
+Self-hosted routers running with the [Free plan](https://www.apollographql.com/pricing) are rate limited to 60 requests per minute. 
```

Comment:
This preposition sounds more natural to me, but it could just be me:

```suggestion
Self-hosted routers running on the [Free plan](https://www.apollographql.com/pricing) are rate limited to 60 requests per minute. 
```